### ISSUE TITLE
dx11 client doesn't work (dx9 works with the generated argument)

### ISSUE THREAD

**Issue by freehaha** [xivapi/ffxiv-launcher#11]
I was trying out this launcher but the generated argument only works with the dx9 client. When I open dx11 client with the exact argument I get an error message saying that my account isn't registered for the game or subscription has run out. I didn't use this launcher before this patch(4.57) so I don't know if it worked before with the same setting but I'm assuming it's either because the token is invalid for a dx11 client or they changed how the dx11 client works.

**Comment by vekien** [comment]
Have you got the latest version?

I use the launcher daily and it is working for me right now.

**Comment by freehaha** [comment]
yes i'm using the master version. However I'm on linux so i had to modify it slightly to pass the arguments to my client, basically this is what it looks like in the end:
```
$WINE "$WINEPREFIX/$XIVPATH/ffxiv_dx11.exe" DEV.UseSqPack=1 DEV.DataPathType=1 DEV.TestSID=xxxx DEV.MaxEntitledExpansionID=2 language=1 SYS.Region=3 ver=2019.04.16.0000.0000
```
dx9 executable(`ffxiv.exe`) works fine so i'm not sure how dx11 is rejecting the arguments.

**Comment by jbalme** [comment]
That means it's probably something Linux specific.

@viion Prepare yourself, you're probably going to get a bunch of tickets from Linux users because the new patch broke the launcher in Wine.

**Comment by freehaha** [comment]
Looks like it's not a problem in the scope of this project, but I guess we can still use it as a dx9 client launcher in the mean time.

**Comment by HereInPlainSight** [comment]
So, very belatedly, this is likely an issue with the fact that Square started detecting Wine and identifying it as a Mac client.

Hiding wine exports is likely to have resolved this issue.
